site/en/r2/tutorials/load_data/numpy.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Why not use Sequential API?</p><p><br></p><p>This is straightforward, so sequential makes more sense here.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I didn't write it, so I can't answer why. 
But I don't mind changing it.

Reply:
Done.